scripts/helper_functions.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
I'm a bit concerened about the order of files. depending on how you run them, they might not have been defined yet. For bash it might be better to define functions in reverse order.

Probably doesn't matter here though, because you don't execute anyting in this file. Just be aware